packages/webdriverio/src/commands/mobile/swipe.ts

```diff
@@ -136,7 +136,7 @@ async function calculateFromTo({
     //    - bottom
     //    - left
     //    of the element. These positions will contain the x and y coordinates on where to put the finger
-    const { x, y, width, height } = await browser.getElementRect(scrollableElement?.elementId)
+    const { x, y, width, height } = await browser.getElementRect(await scrollableElement?.elementId)
```

Comment:
@christian-bromann 

Hmm, might this fix the error @dinh246 was talking about? I first tested it on my machine with the old code and there was no issue. Then after running the adjusted appium-boilerplate code on GHA+Sauce I got the same error with the current `webdriverio`package, see https://github.com/webdriverio/appium-boilerplate/actions/runs/12587209761/job/35082582798#step:9:221

Reply:
Hmm, it might be, it's now also fails on iOS, see https://github.com/webdriverio/appium-boilerplate/actions/runs/12587209746/job/35082583004#step:7:254

This seems to be working on my machine due to the M3 and not on slower machines. So this fix might solve the issue